src/components/AssetSelection/components/AssetChainDropdown/AssetChainDropdown.tsx

```diff
@@ -38,7 +42,6 @@ export const AssetChainDropdown: React.FC<AssetChainDropdownProps> = ({
   buttonProps,
   isLoading,
   isError,
-  ...menuProps
```

Comment:
spread operator causes re-render every frame

Reply:
Fine to remove in this case as we weren't using menu props, but indeed, const reassignment of `menuProps` every render is inherently dangerous, nice catch!

Don't think we should necessarily step away from that pattern, we could always `useMemo` when there's a legit use case for the rest props.

But I do agree it's always better to be fully declarative and pass all props explicitly, that would've avoided this guy here, where `<Menu />` doesn't actually doesn't consume any of said rest props and an effective empty object is passed